src/Lecture1.hs

```diff
@@ -108,7 +120,14 @@ and finds a sum of the numbers inside this string.
 
 The string contains only spaces and/or numbers.
 -}
-strSum str = error "TODO"
+strSum :: [Char] -> Int
+strSum str = go 0 (words str)
+    where
+        go :: Int -> [[Char]] -> Int
+        go result w =
+            if null w
+            then result
+            else go (result + (read (head w))) (tail w)
```

Comment:
There's already a standard function `sum` to find the sum of all elements in the list. So you don't need to write your own function 🙂

If you use standard function `map` to apply `read` to every element of the list, together with `sum` and `words` you'll be able to implement `strSum` as a nice one-liner!